src/p_saveg.c

```diff
@@ -86,7 +86,7 @@ typedef union
 
 inline static void saveg_write_float(float value)
 {
-    fdata_t data = {.f = FLOAT(value)};
+    fdata_t data = {.f = SWAP_FLOAT(value)};
```

Comment:
Are we sure this is completely portable across all possible float implementations? Just a thought, but using 16.16 fixed point math we could still save the position of a several-thousand-seconds long sound with sub-millisecond resolution.

Reply:
I think it's very hard to find a system that isn't IEEE-754 compatible these days, so it should be okay. However, the fixed math version could be simpler (there would be no need to create new `saveg_write_float` functions).

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/2280/commits/09e06188a318ee3572e7f8f982438e721a8f970e